src/service/ResumeSearcher.java

```diff
@@ -0,0 +1,9 @@
+package service;
+
+public class ResumeSearcher implements Searchable {
+
+    @Override
+    public void search() {
```

Comment:
methods like `search`, `find`, `retrieve` etc. should return object which we are finding, and get parameters for searching (e.g. id or some predicate)

Reply:
Fixed